bin/saya/scripts/0_account_setup.sh

```
@@ -0,0 +1,15 @@
+#Set all the environment variables in the script for the account setup.
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)